lightwood/mixer/lightgbm.py

```diff
@@ -65,17 +64,15 @@ def __init__(
         self.supports_proba = dtype_dict[target] in [dtype.binary, dtype.categorical]
         self.stable = True
 
-        # GPU Only available via --install-option=--gpu with opencl-dev and libboost dev (a bunch of them) installed, so let's turn this off for now and we can put it behind some flag later # noqa
-        gpu_works = check_gpu_support()
-        if not gpu_works:
-            self.device = torch.device('cpu')
-            self.device_str = 'cpu'
-            log.warning('LightGBM running on CPU, this somewhat slower than the GPU version, consider using a GPU instead') # noqa
-        else:
-            self.device = torch.device('cuda')
-            self.device_str = 'gpu'
-
-        self.max_bin = 255
+        # Disabled until fix for: https://github.com/microsoft/LightGBM/issues/3339
```

Comment:
Also, please comment on how much the slowdown is between CPU/GPU. Run some datasets and get a rough estimate as a function of dataset size.

If the answer is "this is negligibly different (5-15% slower)" I do not think this is our top priority. 

If this is causing an actual bug in which the mixer is *not* running, then I am ok accepting this PR with the understanding we should find out whether GPU support is actually feasible.

Reply:
> Also, please comment on how much the slowdown is between CPU/GPU. Run some datasets and get a rough estimate as a function of dataset size.

It's hard to get LGBM to work, so it's not available on most machines, hence this PR will not affect them.

I can try running a few tests but that will take a while.

> If this is causing an actual bug in which the mixer is not running, then I am ok accepting this PR with the understanding we should find out whether GPU support is actually feasible.

Yes, that is the issue, but in all honesty I just checked, and if people have lightgbm GPU installed this will still not run, so I guess it doesn't really matter (?)